src/Workspaces/Core/Portable/CodeActions/CodeAction.cs

```diff
@@ -25,6 +25,8 @@ public abstract class CodeAction
         /// </summary>
         public abstract string Title { get; }
 
+        internal virtual string Message => Title;
```

Comment:
We should consider either making this public, so that 3rd parties can also override it, or making it a constructor parameter.